### PR TITLE
Theta quality plots for DT Trigger in DQM (port to 81x)

### PR DESCRIPTION
**Pull request by jfernan2**
Theta quality plots for DT Trigger in DQM
81X port of #14750

### CODE REVIEW

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1
please set the proper title for this PR

**Comment by jfernan2** [comment]
Any news on this?
Thanks

File: EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc [hunk 1]
```diff
@@ -78,13 +78,13 @@ class L1TCaloLayer1RawToDigi : public stream::EDProducer<> {
   virtual void produce(Event&, const EventSetup&) override;
   virtual void endStream() override;
 
-  void makeECalTPGs(uint32_t lPhi, UCTCTP7RawData& ctp7Data, std::unique_ptr<EcalTrigPrimDigiCollection>& ecalTPGs);
+  void makeECalTPGs(uint32_t lPhi, UCTCTP7RawData& ctp7Data, std::auto_ptr<EcalTrigPrimDigiCollection>& ecalTPGs);
```
Comment (dmitrijus) [inline EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc]: These should be unique_ptr, please update so this PR does not change unique_ptr.
I guess you directly copied this file from 80x, from before auto_ptr -> unique_ptr migration happened.

**Comment by davidlange6** [comment]
please test
ping @mulhearn @rekovic

**Comment by dmitrijus** [comment]
The PR title should what changes this PR introduces - "81x port of #14750" does not really tell much.
Please set it to "Theta quality plots for DT Trigger in DQM (port to 81x)"

**Comment by jfernan2** [comment]
Hi, 
This PR is rather old now, is there anything preventing this to be merged?
Thanks in advance

File: EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc (hunk 1 above)
Comment (davidlange6) [inline EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc]: @jfernan2 - this comment needs to be addressed...

**Review by dmitrijus — APPROVED** [review]

**Comment by dmitrijus** [comment]
+1

**Comment by rekovic** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**